test/integration/test_runner.sh

```diff
@@ -12,11 +12,15 @@ function execute() {
 }
 
 # Tests to run. Defaults to all.
-TESTS=${@:-. discovery api mesos/api}
+TESTS=${@:-. compose discovery api mesos/api mesos/compose}
 
 # Generate a temporary binary for the tests.
 export SWARM_BINARY=`mktemp`
 
+# Download docker-compose
+execute time curl -L --silent https://github.com/docker/compose/releases/download/1.4.0/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```

Comment:
Could you make `docker-compose` version configurable?
Like this:
`https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose`

I think it's better for regression across different version of `docker-compose` too.